shell/components/form/LabeledSelect.vue

```diff
@@ -284,6 +285,11 @@ export default {
         <template v-else-if="option.kind === 'divider'">
           <hr />
         </template>
+        <template v-else-if="option.kind === 'link'">
+          <a>
```

Comment:
Could you also please replace the `a` element with a `div` as it's done for the other options?

![Screenshot 2022-10-07 at 12 17 18](https://user-images.githubusercontent.com/5009481/194531354-bb86cd88-1578-4ecb-9d48-de866b41ce7d.png)